src/main/java/org/gridsuite/shortcircuit/server/repositories/ShortCircuitAnalysisResultRepository.java

```diff
@@ -68,11 +68,15 @@ private static FaultResultEntity toMagnitudeFaultResultEntity(FaultResult faultR
 
         double ipMax = Double.NaN;
         double ipMin = Double.NaN;
+        double deltaCurrentIpMin = Double.NaN;
+        double deltaCurrentIpMax = Double.NaN;
         if (shortCircuitLimits != null) {
             ipMax = shortCircuitLimits.getIpMax();
             ipMin = shortCircuitLimits.getIpMin();
+            deltaCurrentIpMin = current - ipMin / 1000;
```

Comment:
ipMax = shortCircuitLimits.getIpMax();
 ipMin = shortCircuitLimits.getIpMin();
 deltaCurrentIpMin = current - ipMin / 1000;

Reply:
it's simpler

Reply:
I keeped ipMax and ipMin without conversion because it's already done in Frontend. But I can do it here.

Reply:
> ipMax = shortCircuitLimits.getIpMax()/ 1000; ipMin = shortCircuitLimits.getIpMin()/ 1000; deltaCurrentIpMin = current - ipMin; deltaCurrentIpMax = current - ipMax;

If I do this, won't we have an inconsistency at the DB values(old values on A and the new values on KA)?

Reply:
Ah yes, you're right. Ok to keep it like this